x-pack/plugins/endpoint/server/config.ts

```diff
@@ -17,6 +17,10 @@ export const EndpointConfigSchema = schema.object({
   endpointResultListDefaultPageSize: schema.number({ defaultValue: 10 }),
   alertResultListDefaultFirstPageIndex: schema.number({ defaultValue: 0 }),
   alertResultListDefaultPageSize: schema.number({ defaultValue: 10 }),
+  alertResultListDefaultFilters: schema.string({ defaultValue: '' }),
```

Comment:
More of a general comment similar to what Rob asked on my PR, do you think we need to give the user the ability to configure these? I'm wondering if we should just make most of these a hardcoded value in the source.

Reply:
@jonathan-buttner No strong feelings... just following the already established pattern, but it's a valid question.